docs/administration/back_office/content_tab_switcher.md

```diff
@@ -29,6 +29,58 @@ ibexa:
 
 ![Meta tab](tab_switcher.png)
 
+
+### Configure Field groups for Meta tab
+
+The default configuration causes that the `ibexa_taxonomy_entry_assignment` Field is always shown in the Meta tab in the Content form. 
+With this new feature, you can indicate what Field types, previously set in the Back Office Content Type, are located in the Meta tab section in the Content form. 
+
+
+
+You can automatically move all Field types from Metadata group to the Meta tab in the Content form.
+To do it, use the following configuration:
+
+```yaml
+    system:
+        admin_group:
+            admin_ui_forms:
+                content_edit:
+                    meta_field_groups_list:
+                       - metadata
+
+```
+
+![Meta tab](tab_switcher_meta.png)
+
+
+
+If you want to automatically move all Field types from Content group to the Meta tab, use the following configuration:
```

Comment:
Not sure if this is a good example because `Content` field group is the default one that shouldn't really be messed with. In our case only have 2 field groups (Content and Metadata) by default, the example would be better if it mentioned new field group instead.

Reply:
I removed this section to avoid ambiguities.